src/Uno.UI/Microsoft/UI/Xaml/Controls/Breadcrumb/BreadcrumbBar.cs

```diff
@@ -720,4 +733,23 @@ private void OnAccessKeyInvoked(UIElement sender, AccessKeyInvokedEventArgs args
 		}
 	}
 
+#if HAS_UNO
+	private bool _isUnloaded = false;
```

Comment:
Is it not possible to use `IsLoaded` ?

Reply:
In this case I need to know in `Loaded` handler that the control was `Unloaded` before. If I used `IsLoaded`, it would always be `true` in `Loaded` event, even though there was no unload before it (therefore repeating `OnApplyTemplate` for no reason).